ModularTegustation/modular_abnormalities/zayin/a_quiet_day/code/quiet_day.dm

```diff
@@ -5,7 +5,7 @@
 /mob/living/simple_animal/hostile/abnormality/quiet_day
 	name = "A Quiet Day"
 	desc = "An old weather damaged bench, it feels oddly nostalgic to you. Like a spring day at the side of a lake."
-	icon = 'ModularTegustation/Teguicons/48x48.dmi'
+	icon = 'ModularTegustation/modular_abnormalities/zayin/a_quiet_day/icons/48x48.dmi'
```

Comment:
Okay i thought at first you were doing like icons generalized to all zayins instead of a dmi for each abnormality. I worry that having a non generalized dmi for abnormalities would, similar to the sound effects, result in duplicate icons in two separate files. I honestly thought the file would be Zayin -> Icons, Sound, Code -> individual abnormality code. But since i dont truely understand why the elder coders made the code so decentralized like this im not sure if my suggestion is even good.

Reply:
I do feel like it would be better to have a generalized zayin_32x32.dmi instead of 8 separate 32x32.dmi's.

Reply:
because thats how modular files work, each of the zayin's counts as a seperate modular file.
This makes it so icon conflicts are basically impossible, and you dont have to look at anything except the modular file itself for all that the abnormality needs to function

Reply:
the only way to get an icon conflict in this way, is for 2 seperate spriters to decide that they are changing the same exact abnormality's sprite at the same exact moment.
Or for 1 spriter to change the main sprite, and a coder to add a new mechanic to the abnormality that uses a second sprite (for example adding a breaching sprite to an abnormality that does not have one)